### PR TITLE
Update wording for "section identifiable not found"

### PR DESCRIPTION
**Pull request by anistouri**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

File: iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateCouplingDevice.java [hunk 1]
```diff
@@ -176,7 +176,7 @@ private boolean failBbs(Identifiable<?> bbs1, Identifiable<?> bbs2, Reporter rep
     }
 
     private static void busOrBbsDoesNotExist(String bbsId, Reporter reporter, boolean throwException) {
-        LOGGER.error("Identifiable {} not found.", bbsId);
+        LOGGER.error("Section identifiable {} not found. Bus-tie or omnibus not created.", bbsId);
```
[the remark this answers is not in the document]
Comment (geofjamg) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateCouplingDevice.java:179]: So it should be "Bus or busbar section {] not found"

[the remark this answers is not in the document]
Comment (geofjamg) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateCouplingDevice.java:179]: is omnibus a correct english word?

[the remark this answers is not in the document]
Comment (anistouri) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateCouplingDevice.java]: I changed the wording following the PO request

**Review by FranckLecuyer — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

File: iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/util/ModificationReports.java [hunk 2]
```diff
@@ -336,7 +336,7 @@ public static void voltageLevelRemovingEquipmentsLeftReport(Reporter reporter, S
     public static void notFoundIdentifiableReport(Reporter reporter, String identifiableId) {
         reporter.report(Report.builder()
```
Comment (annetill) [inline iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/util/ModificationReports.java:337]: This method is too generic to have a message linked to couplers.

**Review by annetill — CHANGES_REQUESTED** [review]

File: iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/util/ModificationReports.java (hunk 2 above)
Comment (anistouri) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/util/ModificationReports.java:337]: I have created a new method to handle that

**Comment by flo-dup** [comment]
@anistouri Regarding last commit I added: why did you solve the problem only at one place? `notFoundIdentifiableReport` was called at 3 different places. Instead of solving the problem 3 times, just solve it once.

**Review by flo-dup — APPROVED** [review]

**Pull request merged**